R/req-retries.R

```diff
@@ -110,15 +110,29 @@ retry_backoff <- function(req, i) {
   req_policy_call(req, "retry_backoff", list(i), default = backoff_default)
 }
 
-retry_after <- function(req, resp, i) {
+retry_after <- function(req, resp, i, error_call = caller_env()) {
   after <- req_policy_call(req, "retry_after", list(resp), default = resp_retry_after)
+
+  # TODO: apply this idea to all callbacks
+  if (!is_number_or_na(after)) {
+    not <- obj_type_friendly(after)
+    cli::cli_abort(
+      "The {.code after} callback to {.fn req_retry} must return a single number or NA, not {not}.",
```

Comment:
Out of curiosity, why `not`, vs this?
```suggestion
      "The {.code after} callback to {.fn req_retry} must return a single number or NA, not {.obj_type_friendly {after}}.",
```

Reply:
Just to make the lines shorter.

Reply:
Aha! I thought that might be it, but the GitHub web UI made it not look THAT long. Glad to know I wasn't missing something!